common/python/database/test_lmdb_app.py

```diff
@@ -66,6 +68,71 @@ def test_get_value2(self):
         self.assertEqual(get_value_result2, self.value2,
                          "get_value Retrieved incorrect value")
 
+    def test_csv_append(self):
```

Comment:
Can you add this test as part of CI run(tools/run_tests.sh) so that we will better coverage and also protect the code.

Reply:
Makes total sense @Ram-srini. As of now I have added cases and run them manually. Will take the automation in a separate PR.